profiles/linuxdotexe.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/74004229/147823124-c920c106-4eb3-4736-a2f3-da9398dc54ba.png",
+  "issueId": 948,
+  "name": "sai nivas mangu",
```

Comment:
```suggestion
  "name": "Sai Nivas Mangu",
```

Reply:
Made the required changes. Please verify.